openmetadata-ui/pom.xml

```diff
@@ -95,7 +95,7 @@
       <plugin>
         <groupId>com.github.eirslett</groupId>
         <artifactId>frontend-maven-plugin</artifactId>
-        <version>1.11.0</version>
+        <version>1.12.1</version>
```

Comment:
@pmbrull , can we revert these file changes. because it might break UI build.

Reply:
ups didn't mean to push this, good catch. I just need it in local to make it run. Reverting